cdap-docs/tracker-manual/source/index.rst

```diff
@@ -165,19 +165,16 @@ in the ``cdap-site.xml`` as described above, need to follow these steps:
 
 .. highlight:: json  
 
-- Create an application configuration file (``appconfig.txt``) that contains the Kafka
-  Audit Log reader configuration (the property ``auditLogKafkaConfig``). It is the Kafka
-  Consumer Flowlet configuration information. For example::
+- Create an application configuration file (``appconfig.txt``) that contains the
+  Audit Log reader configuration (the property ``auditLogConfig``). For example::
     
     {
       "config": {
-        "auditLogKafkaConfig": {
-          "zookeeperString": "<host>:<port>/cdap/kafka"
+        "auditLogConfig" : {
```

Comment:
Should we leave zookeeperString property in there (or mention that it is required in distributed mode?) ?